### DISCUSSION TITLE
Support for Multiple Soundbars

### DISCUSSION THREAD

**Discussion by royf007** [PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar#61] [Q&A] [answered]
This a a really great integration. I have 2 soundbars, and wanting to know how I can use more than 1 soundbar with this integration? It looks like I can only specify 1 device ID.

**Comment by PiotrMachowski** [comment, marked as answer]
I think it should be possible without any issues. Example config:

```yaml
media_player:
  - platform: smartthings_soundbar
    name: Soundbar 1
    api_key: "YOUR API KEY"
    device_id: "YOUR DEVICE ID 1"
  - platform: smartthings_soundbar
    name: Soundbar 2
    api_key: "YOUR API KEY"
    device_id: "YOUR DEVICE ID 2"

**Comment by royf007** [comment]
This worked perfectly!! I'm just getting used to Home Assistant.

Thank you again! 😀
Roy.

**Reply by PiotrMachowski** [reply]
Great, I'm glad it works for you 👍
